web-api/src/notifications/retrySendNotificationToConnections.ts

```diff
@@ -1,4 +1,6 @@
 // eslint-disable-next-line spellcheck/spell-checker
+import { Connection } from '@web-api/notifications/sendNotificationToConnection';
```

Comment:
Just curious, why were the notification functions moved to `web-api/notifications` instead of `web-api/src/notifications`?

Reply:
Those functions are in web-api/src. I feel like maybe we should change it to be less confusing eventually.

The alias @web-api/ is short for ./web-api/src/

Reply:
Oh I was reading the file explorer in the pull request view wrong. Thanks!